.travis.yml

```diff
@@ -22,17 +21,23 @@ install:
   - pip install -r requirements.txt
   - pip install codecov
   - pip install pytest-cov
+  - pip install mypy
 
 env:
   - BB_UNIONFS_ENABLE=false BB_PLUGINS_EXPERIMENTS='["benchbuild.experiments.raw", "benchbuild.experiments.empty"]'
 
 script:
-  - coverage run -a `which benchbuild` bootstrap -s
-  - pytest --cov=./ benchbuild tests
-  - coverage run -a `which benchbuild` config view
-  - coverage run -a `which benchbuild` project view
-  - coverage run -a `which benchbuild` experiment view
-  - coverage run -a `which benchbuild` -vvvvv run --full test
+  - coverage run -p `which benchbuild` bootstrap -s
+  - pytest --cov-config=./.coveragerc --cov=./ benchbuild tests
+  - coverage run -p `which benchbuild` config view
+  - coverage run -p `which benchbuild` project view
+  - coverage run -p `which benchbuild` experiment view
+  - coverage run -p `which benchbuild` -vvvvv run --full test
+  - coverage combine -a
+  - coverage report
+  - mypy --strict benchbuild
```

Comment:
that works on full BB?

Reply:
mypy? No. Errors out like crazy. Mhh you're right, maybe i just let it run but don't fail the build.
I'll have another look at this.

Reply:
You can also try non strict mode or tell mypy to only check the diff, but both alternatives assume you have some level of type annotations in the code base.